docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+# must accept EULA the first time that the image is started
+# docker run splunk/enterprise:6.5.2
+# Options on how to review the EULA and accept it: 
+# 1. docker run -it splunk/enterprisetrial:6.5.2
+# 2. Add the following environment variable: SPLUNK_START_ARGS=--accept-license
+# e.g., docker run -e "SPLUNK_START_ARGS=--accept-license" splunk/enterprisetrial 
```

Comment:
Minor consistency concerns about this compose file:
- It uses "enterprisetrial" for a container name where none of the other examples seem to do so.
- It uses port 9000 for the external port map, again inconsistent with the others.